readthedocs/core/management/commands/update_docker.py

```diff
@@ -20,4 +20,6 @@ def handle(self, *args, **options):
                 project_data = api.project(slug).get()
                 p = tasks.make_api_project(project_data)
                 log.info("Building %s" % p)
-                tasks.update_docs(pk=p.pk, docker=True)
+                # TODO just move this to update_api.py
```

Comment:
We should decide what management commands are worth sticking around. Either way, build vs docker build should just be a parameter to the build comand.